### ISSUE TITLE
covidcast_cor issues warnings with dplyr 1.0.0

### ISSUE THREAD

**Issue by capnrefsmmat** [cmu-delphi/covidcast#338]
With dplyr 1.0.0, I get warnings like these when running the tests:

```
Warning (test-cor.R:115:3): lags are applied in correct temporal direction
The `.dots` argument of `group_by()` is deprecated as of lifecycle 1.0.0.
Backtrace:
  1. testthat::expect_equal(covidcast_cor(foo, bar, dt_y = 4), expected) test-cor.R:115:2
  8. dplyr:::group_by.data.frame(., .dots = as.symbol(by))
  9. dplyr::group_by_prepare(.data, ..., .add = .add)
 10. lifecycle::deprecate_soft("1.0.0", "group_by(.dots = )")
```

As it says, `.dots` is deprecated (and [no longer documented](https://dplyr.tidyverse.org/reference/group_by.html)), so we should switch the code to use the other symbol/quoting features in dplyr so we can turn the string into a symbol it accepts.

**Comment by statsmaths** [comment]
I am not getting this warning with dplyr_1.0.5, testthat_3.0.0, and the current r-pkg-devel. Was this already fixed but just not closed?

**Comment by capnrefsmmat** [comment]
I guess I fixed this in #395 and forgot to close this.
